runner/src/server/forms/components.json

```diff
@@ -11,109 +11,91 @@
           "name": "textField",
           "title": "Text field",
           "hint": "Help text",
-          "options": {},
```

Comment:
why did you take out these options?

Reply:
please don't remove options from the schema. the required flags live there! But the corrections to the list definitions are correct.

any changes to schema I think I'd like to see an ADR or explanation in your PR please

Reply:
sorry @jenbutongit, must of gotten my wires crossed. I have added back the options and kept the correction to lists .